grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/plan/SimplePlanner.java

```diff
@@ -114,16 +110,4 @@
      * NB: atom satisfying this criterion should be resolved last
      */
     public static final int COMPARISON_VARIABLE_VALUE_PREDICATE = - 1000;
-
-    /**
-     * @param query for which the plan should be constructed
-     * @return list of atoms in order they should be resolved using {@link GraqlTraversal}.
-     */
-    static public ImmutableList<Atom> plan(ReasonerQueryImpl query){
-        return ImmutableList.<Atom>builder().addAll(
-                query.selectAtoms().stream()
-                        .sorted(Comparator.comparing(at -> -at.baseResolutionPriority()))
-                        .iterator())
-                .build();
-    }
```

Comment:
this our old planner, it is not used in the code however it is useful for debugging, it uses the priority function from `Atom`